systemtest/approvals/TestIntake/Errors.approved.json

```diff
@@ -319,10 +319,6 @@
                 "pid": 1234,
                 "title": "node"
             },
-            "processor": {
-                "event": "error",
-                "name": "error"
```

Comment:
How come these aren't being set to their `constant_keyword` value?

Reply:
That's because we're looking at `_source`, and we don't add the fields to `_source` any more. We should probably be comparing `fields`. If we do that, it will require updating all approvals, as `fields` field names flattened, and the values are always arrays.

Reply:
ah I see. Thank you for the clarification.

Reply:
:+1: I'll open an issue about this tomorrow, and we can figure out how best to do that.